blaserunning-and-shame/README.md

```diff
@@ -0,0 +1,56 @@
+**DRAFT**
+
+# Effects of the "Blaserunning" decree on shame
+
+[Society for Internet Blaseball Research](https://github.com/Society-for-Internet-Blaseball-Research)
+
+Authors: iliana etaoin (she/her)
+
+**Abstract**: We consider whether the "Blaserunning" decree in the ILB Season 3 election, which awards 0.1 runs for a team that successfully steals a base, impacts the number of games resulting in shame.
+We apply this rule to past games and find a 21.6% decrease in games where the away team is shamed.
+
+## Shame and Blaserunning
+
+In [blaseball](https://blaseball.com), the *shame phase* was the first major difference between its ruleset and baseball's.
+The [Blaseball Official Rulebook](https://blaseball.com/thebook) (It is Forbidden) defines it:
+
+> Shame Phase: If the home team scores the winning run in the bottom of the final inning, the away team must complete the game in shame, despite being mathematically eliminated.
+
+For example, if the Magic is playing at the Tacos, and the score is tied entering the bottom of the 9th inning, the Tacos can win the game by scoring a run, thus shaming the Magic.
```

Comment:
"is playing at", is that a sports-specific phrase? it sounds like it could be right but it also sounds like it could be a typo.

Reply:
remove the comma after "Tacos"